docs/structure-samples/javascript/javascript.md

```diff
@@ -46,8 +46,11 @@ The following guidelines and the folders included, represent the conventional st
 ├── * docs/                           
 │   └── ...                         
 │
-├── package.json                    
-├── README.md                       
+├── .env.example # Store project related env vars example
```

Comment:
Interesting, I only have .env.example if I also support a non-azd deployment path. Otherwise, I assume they'll use azd to create the environments, and it can be confusing to provide both, I think.

Reply:
It's a good point. I'll rethink how we document this in code without creating confusion.